src/javascript/Shopper/Public/ShopperLanding.tsx

```diff
@@ -0,0 +1,18 @@
+import { FC } from 'react'
+
+import ShopNav from '../ShopNav'
+
+const ShopperLanding: FC = () => {
+  return (
+    <main>
+      <ShopNav />
```

Comment:
If you place `ShopNav` in the Layout component, it will be visible in every route that uses the layout without having the import it into every file.